README/Posts-Projects.md

```diff
@@ -0,0 +1,51 @@
+#### 1. Posts Creation 
+
+If you want to create a post you should create a file in _posts, the file name structure need to be YYYY-MM-DD-name-of-the-post.md
+
+![posts](./img-posts-projects/posts.png)
+
+This is what  we need to add to our new post:
+
+- **lng_pair:** name_id (an id to link the same post in another language, you will need to use the same, at the post to link in another language)
+- **title:** title (is the title it will appear into the post)
+- **author:** author_name (the name author of this post, if you don’t use this, it will appear one automatically)
+- **category:** name (a category to vinculate the post, if it doesn’t exists it creates automatically)
```

Comment:
There are a few things ehre:
1. It's better to use `link` instead of `vinculate`
2. For apostrophe use ' instead of `
3. It's `exist` instead of `exists`